python/tidy/servo_tidy/tidy.py

```diff
@@ -19,6 +19,56 @@
 import sys
 from licenseck import MPL, APACHE, COPYRIGHT, licenses_toml, licenses_dep_toml
 import colorama
+import toml
+
+CONFIG_FILE_PATH = os.path.join(".", "servo-tidy.toml")
+
+# Default configs
+config = {
+    "skip-check-length": False,
+    "skip-check-licenses": False,
+    "ignore": {
+        # Ignore hidden files by default
+        "files": ["./.", CONFIG_FILE_PATH],
+        # Ignore hidden directories by default
+        "directories": ["./."],
+        "packages": [],
+    }
+}
+
+# Load configs from servo-tidy.toml
+if os.path.exists(CONFIG_FILE_PATH):
```

Comment:
It would be better if we do all this in `check_config`. Or better yet, wrap this inside a function and call it after checking the config file. That way, we'll have to check whether the config exists only once, and we won't be needing a control jump for tests.